examples/docs_snippets_crag/setup.py

```diff
@@ -16,5 +16,5 @@
         ],
         packages=find_packages(exclude=["test"]),
         install_requires=["dagster"],
-        extras_require={"full": ["seaborn", "scikit-learn"]},
+        extras_require={"full": ["seaborn", "scikit-learn", "moto==1.3.16"]},
```

Comment:
is there a reason we hard pin `moto` here?

Reply:
no...I think this was a copy paste from a different setup file